devtools/types/src/index.ts

```diff
@@ -0,0 +1,239 @@
+import type { Flow } from "@player-ui/types";
+
+export interface BaseEvent<T extends string, P> {
```

Comment:
Should `P` just have a default value of `null`?

Reply:
Updated.